src/SourceBuild/content/eng/allowed-vmr-binaries.txt

```diff
@@ -118,7 +118,7 @@ src/sdk/test/TestAssets/TestProjects/**/*.dat
 src/sdk/test/TestAssets/TestProjects/**/*.cache
 src/sdk/test/TestAssets/TestProjects/**/*.tlb
 src/sdk/test/TestAssets/TestPackages/dotnet-new/nupkg_templates/*
-src/sdk/test/Microsoft.DotNet.ShellShim.Tests/WpfBinaryTestAsssets/*.dll
+src/sdk/test/dotnet.Tests/ShellShimTests/WpfBinaryTestAssets/testwpf.dll
```

Comment:
I think we should preserve the wildcard.  It allows for more flexibility and reduces the maintenance if the test is refactored to rename the binary or add others.  Given this is a "test assets" folder, this feels reasonable.

```suggestion
src/sdk/test/dotnet.Tests/ShellShimTests/WpfBinaryTestAssets/*.dll
```